src/libraries/System.Net.Http/src/System/Net/Http/HttpClient.cs

```diff
@@ -2,9 +2,9 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
```

Comment:
probably should be removed

Reply:
It's used for a `[NotNull]` attribute in one place.